tests/test_lsst_science_pipeline.py

```diff
@@ -29,6 +29,7 @@ def pes_lens_instance():
             variability_model="sinusoidal",
             kwargs_variability={"amp", "freq"},
             cosmo=cosmo,
+            lightcurve_time=np.linspace(0, np.pi, 100),
```

Comment:
I don't understand why lightcurve_time is linear between zero and pi

Reply:
Here, I am using sinusoidal function as a lightcurve for testing purpose. So, I just provided values between 0 to pi as a time array. This is a old test function (we are not writing any test function for lsst_science_pipeline. These are the only function we can test). If we use actual supernovae or quasar lightcurves, then we should provide days as observation time.